sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -136,7 +136,7 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     // some expression is reusing variable names across different instances.
     // This behavior is tested in ExpressionEvalHelperSuite.
     val plan = generateProject(
-      GenerateUnsafeProjection.generate(
+      UnsafeProjection.create(
```

Comment:
yea.

`GenerateUnsafeProjection.generate` here was not able to use unsafe-backed data structure because it's `Create*Struct`.
`UnsafeProjection.create`, however, does use unsafe-backed data structure (UnsafeRow, UnsafeArrayData, ...) so that this test is valid.